core/templates/pages/topic-editor-page/editor-tab/topic-editor-stories-list.component.html

```diff
@@ -49,20 +111,41 @@
     width: 75%;
   }
 
+  .story-list-item {
+    background: #fff;
+    font-size: 14px;
+    margin: 20px 7.5px 0 12.5%;
+    padding: 10px 20px;
+    width: 75%;
+  }
+
+  .story-list-item td {
+    padding: 2%;
+  }
+
   .dashboard-table {
     font-size: 15px;
     margin-top: 2vh;
     width: 100%;
   }
 
-  .dashboard-table .dashboard-table-headings {
+  .dashboard-table .dashboard-table-headings, .story-dashboard-table-headings {
     border-bottom: 2px solid #000;
     margin: 25px 0;
   }
 
-  .dashboard-table .dashboard-table-headings tr {
-    font-size: 16px;
-    font-weight: bold;
+  .story-dashboard-table-headings p {
+    font-size: 13px;
+    word-spacing: 100vw;
+  }
+
+  .story-dashboard-table td {
+    border: 0.5px solid #dbdbdb;
+  }
+
+  .story-dashboard-table .delete-story-icon {
+    background-color: white !important;
```

Comment:
Hey @Kej-r03 I think the `!important` is not really recommended. Can we find an alternative here ? Thanks!

Reply:
Changed